cmd/tailscale/cli/serve_v2.go

```diff
@@ -128,7 +128,7 @@ func newServeV2Command(e *serveEnv, subcmd serveMode) *ffcli.Command {
 			fs.StringVar(&e.http, "http", "", "Expose an HTTP server at the specified port")
 			fs.StringVar(&e.tcp, "tcp", "", "Expose a TCP forwarder to forward raw TCP packets at the specified port")
 			fs.StringVar(&e.tlsTerminatedTCP, "tls-terminated-tcp", "", "Expose a TCP forwarder to forward TLS-terminated TCP packets at the specified port")
-
+			fs.BoolVar(&e.yes, "yes", false, "update without interactive prompts")
```

Comment:
Let's either uppercase the first character to match the others, or lowercase the rest.